Prevents:
JS ERROR: Gio.IOErrorEnum: Could not connect: No such file or directory
get@resource:///org/cinnamon/cjs/modules/core/overrides/Gio.js:509:28
populateLayouts@/data/data/com.termux/files/usr/share/cinnamon/js/ui/keyboardManager.js:367:9

--- a/js/ui/keyboardManager.js
+++ b/js/ui/keyboardManager.js
@@ -425,7 +425,7 @@ var InputSourceSettings = class {
         this._settings.set_value(KEY_INPUT_SOURCES, sources);
 
         let locale1 = new Locale1Settings();
-        locale1.populateLayouts();
+        // locale1.populateLayouts();
     }
 
     _getSourcesList(key) {
